app/templates/upgrades.hbs

```diff
@@ -1 +1,30 @@
+<div class="row mx-2 my-2">
+  {{#each displayedUpgrades as |upgrade|}}
+    <div class="col-sm-3">
+      <div class="card">
+        <div class="card-header">
+          {{upgrade.name}}
+        </div>
+        <div class="card-body">
+          <p class="card-text">{{upgrade.description}}</p>
+          <button type="button" class="btn btn-primary" disabled={{upgrade.cannotBuy}} onclick={{action "buy" upgrade}}>
+            Buy for
+            {{#if upgrade.manaCost}}
+              <FaIcon @icon="dot-circle" /> {{upgrade.manaCost}}
+            {{/if}}
+            {{#if upgrade.cultureCost}}
+              <FaIcon @icon="dot-circle" /> {{upgrade.cultureCost}}
+            {{/if}}
+            {{#if upgrade.moneyCost}}
+              <FaIcon @icon="dot-circle" /> {{upgrade.moneyCost}}
+            {{/if}}
+            {{#if upgrade.scienceCost}}
+              <FaIcon @icon="dot-circle" /> {{upgrade.scienceCost}}
```

Comment:
Icons are wrong!

Note: Maybe make a todo item for making "something" (a component? helper?) to avoid redefining the icon everywhere.